packages/components/src/tabs/tablist.tsx

```diff
@@ -112,9 +112,15 @@ export const TabList = forwardRef<
 		<Ariakit.TabList
 			ref={ refs }
 			store={ store }
-			render={ <TabListWrapper /> }
+			render={ ( props ) => (
+				<TabListWrapper
+					{ ...props }
+					// Fallback to -1 to prevent browsers from making the tablist
+					// tabbable when it is a scrolling container.
+					tabIndex={ props.tabIndex ?? -1 }
+				/>
+			) }
```

Comment:
Is this something we're interested to have a test for?

Reply:
Sounds good, I'll add one